api/grpc/instances/instances.proto

```diff
@@ -23,12 +23,12 @@ message Meta {
     }
 }
 message NginxMeta {
-    string loadable_modules = 1;
```

Comment:
not sure about removing these?

Reply:
we weren't populating them at the moment so I said I'd remove them for the time being until we start populating them with values. I can add them back in if you want.